services/common/src/main/cspace/config/services/tenants/pahma/pahma-tenant-bindings.delta.xml

```diff
@@ -41,18 +41,6 @@
 						<service:key>sqlScriptName</service:key>
 						<service:value>grant_pahma_reporters_privileges.sql</service:value>
 					</service:property>
```

Comment:
In lines 44-55, you're removing what appears to be three function definitions from PAHMA's tenancy.  Is this really your intent?

Reply:
Yes, those functions were installed long ago when we made the UOC reports.  There's no reason they need to continue to be created/recreated.  They exist in Prod.

Reply:
Lam, when I run a "clean" build of the PAHMA sources, I should get ALL functions currently installed in PROD/production.

Reply:
added PAHMA UOC functions back to runsqlscripts; will create new Jira to update runsqlscripts to build deployment-specific objects.